### ISSUE TITLE
Docker Compose example

### ISSUE THREAD

**Issue by erikthered** [foobaar/kafka-offset-manager#4]
Might be nice to be able to setup a test environment easily.

**Comment by foobaar** [comment]
Yeah, I'll put together a simple single instance kafka and the app together to test it out easily.
 Seed kafka with some test data also..
